static/js/modules/profileStats.js

```diff
@@ -0,0 +1,90 @@
+import { serverData } from "./serverData.js";
+
+var profileStatsTable = {
+
+	data: function () {
+        return {
+            username:"",
+            basic_stats: {
```

Comment:
inconsistency with variable/field names, I think we usually use camelCase for JS

Reply:
Fixed